### ISSUE TITLE
Support "relative steps" in X-axis

### ISSUE THREAD

**Issue by yueqiw** [tensorflow/tensorboard#6969]
Tensorboard has 3 options for X-axis: `Step`, `Relative`, `Wall`. The `Relative` option allows aligning multiple traces that start at different times. 

However, currently we cannot align multiple traces by start step (i.e. `Relative Step`). This feature can be useful because sometimes we have multiple finetuning models that start at different pretraining steps. Without aligning by start step it's hard to visually compare them in the same graph (the only way is to export the data and plot them in colab).

The goal of this FR is to implement `Relative Step`.
